### PR TITLE
Add instructions for PyPI package hash verification

### PR DESCRIPTION
**Pull request by i-oden**
## **1. This PR contains the following changes...**

As a result of [this decision](https://github.com/ScilifelabDataCentre/dds_web/pull/1359), this PR adds instructions to the documentation regarding hash verification.

## **2. The following additional changes are required for this to work**

_X_

## **3. The PR fixes the following GitHub issue / Jira task**

- [ ] GitHub issue (link):
- [x] Jira task (ID, `DDS-xxxx`): DDS-1457
- [ ] The PR does not fix a specific GitHub issue or Jira task

## **4. What _type of change(s)_ does the PR contain?**

- [ ] New feature
  - [ ] Breaking: _Please describe the reason for the break and how we can fix it._
  - [ ] Non-breaking
- [ ] Bug fix
  - [ ] Breaking: _Please describe the reason for the break and how we can fix it._
  - [ ] Non-breaking
- [ ] Security Alert fix
- [x] Documentation
- [ ] Tests **(only)**
- [ ] Workflow

### **Always**

- [Changelog](../CHANGELOG.md)
  - [x] Added
  - [ ] Not needed (E.g. PR contains _only_ tests)
- Rebase / Update / Merge _from_ base branch (the branch from which the current is forked)
  - [ ] Done
  - [x] Not needed
- Blocking PRs
  - [ ] Merged
  - [x] No blocking PRs
- PR to `master` branch
  - [ ] Yes: Go to the section [PR to master](#pr-to-master)
  - [x] No

## **6. Actions / Scans**

- **Black**: Python code formatter. Does not execute. Only tests.
  Run `black .` locally to execute formatting.
  - [x] Passed
- **Prettier**: General code formatter. Our use case: MD and yaml mainly.
  Run `npx prettier --write .` locally to execute formatting.
  - [x] Passed
- **Tests**: Pytest to verify that functionality works as expected.
  - [ ] New tests added
  - [x] No new tests
  - [ ] Passed
- **TestPyPi**: Build CLI and publish to TestPyPi in order to verify before release.
  - [x] Passed
- **CodeQL**: Scan for security vulnerabilities, bugs, errors
  - [ ] New alerts: _Go through them and either fix, dismiss och ignore. Add reasoning in items below._
  - [ ] Alerts fixed: _What?_
  - [ ] Alerts ignored / dismissed: _Why?_
  - [x] Passed
- **Trivy**: Security scanner
  - [ ] New alerts: _Go through them and either fix, dismiss och ignore. Add reasoning in items below._
  - [ ] Alerts fixed: _What?_
  - [ ] Alerts ignored / dismissed: _Why?_
  - [x] Passed
- **Snyk**: Security scanner
  - [ ] New alerts: _Go through them and either fix, dismiss och ignore. Add reasoning in items below._
  - [ ] Alerts fixed: _What?_
  - [ ] Alerts ignored / dismissed: _Why?_
  - [x] Passed

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -255,3 +256,58 @@ Bianca
...
    * Bianca user guide: https://www.uppmax.uu.se/support/user-guides/bianca-user-guide/
    * Transit user guide: https://www.uppmax.uu.se/support/user-guides/transit-user-guide/
+
+
+.. _verify-integrity: 
+
+Verify the package integrity prior to installing it
+====================================================
+
+This installation guide is for those that wish to verify that the ``dds-cli`` package published on PyPI is identical to the one you have locally. Note that the hashes used to verify this are generated by PyPI itself, not by the Data Centre.
```
Comment (i-oden) [inline docs/installation.rst:266]:
```suggestion
This installation guide is for those that want to verify that the ``dds-cli`` package published on PyPI is identical to the one you have locally. Note that the hashes used to verify this are generated by PyPI itself, not by the Data Centre. 
```

File: docs/installation.rst [hunk 2]
```diff
@@ -255,3 +256,58 @@ Bianca
...
+   .. code-block:: bash
+      
+      # "--dest dds-downloaded" tells pip to put the downloaded files in the directory "dds-downloaded"
+      pip download dds-cli --dest dds-downloaded
+
+3. Generate hash for the dds-cli file by running
+
+   .. code-block:: bash
+
+      # [VERSION] should be replaced by the version you have downloaded from PyPI. When in doubt, simply type dds-downloaded/dds_cli and press tab; The path to the file will be filled in automatically.
```
Comment (i-oden) [inline docs/installation.rst:280]:
```suggestion
      # <VERSION> should be replaced by the version you have downloaded from PyPI. When in doubt, simply type dds-downloaded/dds_cli and press tab; The path to the file will be filled in automatically.
```

File: docs/installation.rst [hunk 3]
```diff
@@ -255,3 +256,58 @@ Bianca
...
+
+      # <VERSION> should be replaced by the version you have downloaded from PyPI. When in doubt, simply type dds-downloaded/dds_cli and press tab; The path to the file will be filled in automatically.
+      pip hash dds-downloaded/dds_cli-<VERSION>-py3-none-any.whl
+      # Example output:
+      # dds_cli-<VERSION>-py3-none-any.whl:
+      # --hash=sha256:8ba6495b73d759e96c35652273cf4e4158acba02f1cf64f012cc67cf2e346cae
+
+4. Open a browser and go to the PyPI `"Download files" page <https://pypi.org/project/dds-cli/#files>`_
+
+   1. In the "Source Distribution" section, click "view hashes"
```
Comment (valyo) [inline docs/installation.rst:288]: `Source Distribution`
should be
`Built Distribution`
I think

**Review by valyo — APPROVED** [review]

File: docs/installation.rst (hunk 3 above)
Comment (i-oden) [inline docs/installation.rst:288]:
```suggestion
   1. In the "Built Distribution" section, click "view hashes"
```

**Pull request merged**